### ISSUE TITLE
How hard to make it work front end side?

### ISSUE THREAD

**Issue by hugoroussel** [rckprtr/pumpdotfun-sdk#59]
Would be interested to run this in a webapp with react and phantom

**Comment by thefuzzycreature** [comment]
i think the frontend design is whats hard to make. Its pretty easy to hookup, you just pass in the values. create/buy/sell etc

**Comment by unitydevadtest** [comment]
why? can't you just use pumpfun?

**Comment by saraspaudel** [comment]
No, it shouldn't be hard, you can use the SDK in the frontend too but you have to build the transaction with the wallet provider instead
